layers/meta-balena-5x-owa/recipes-core/switchgsm/files/switchgsm.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Systemd services used for powering up GSM module on boot using Owasys tools.
+Before=NetworkManager.service
+After=pmsrv.service
+Wants=pmsrv.service
+
+[Service]
+Type=oneshot
+RestartSec=10
+Restart=on-failure
+StartLimitBurst= 2
+ExecStart=/usr/bin/Switch_GSM 1
```

Comment:
This is already done in `resin-init` so the `switchgsm` recipe can be removed.

Reply:
Moved to [PR 28](https://github.com/balena-os/balena-owa5x/pull/28)